.bundle/config

```diff
@@ -1,5 +1,5 @@
 ---
 BUNDLE_RETRY: "3"
 BUNDLE_PATH: "vendor/bundle"
-BUNDLE_JOBS: "3"
+BUNDLE_JOBS: "16"
```

Comment:
That's quite the bump!

Just curious: where does 16 come from? Why not a different value?

Reply:
tbh I have no idea. It suddenly started to change that value on its own when I did a `bundle install`, I didn't change it myself, so I think it's just the new default value for more recent `bundler` versions (I'm on `bundler 2.2.19`)

Reply:
Interesting 🤔 

From the [docs](https://github.com/rubygems/rubygems/blob/faa8cf6cccd2ae18eba7191618ff639cc0ddfd7c/bundler/lib/bundler/man/bundle-config.1.ronn):

> jobs (`BUNDLE_JOBS`): The number of gems Bundler can install in parallel. Defaults to 1 on Windows, and to the the number of processors on other platforms.

There must be some bug, unless you are on some [new secret hardware](https://arstechnica.com/gadgets/2021/05/apple-planning-macbook-pro-with-16-core-gpu-mac-pro-with-128-gpu-cores/)

Reply:
😄 if only 😂

Fwiw this change made by `bundle install` when I run it on my machine happens apparently everywhere (ie on all repositories, not just this one). At least that's far from the only time I see it changing that value (and why I started committing it otherwise it kept changing it)